Rakefile

```diff
@@ -4,9 +4,14 @@ require 'html-proofer'
 
 task :test do
   sh 'bundle exec jekyll build'
-  HTMLProofer.check_directory('./_site/',
-                              check_html: true,
-                              hydra: { max_concurrency: 10 },
-                              url_ignore: [%r{https://developer.github.com}, %r{https://docs.github.com}, %r{https://help.github.com}],
-                              http_status_ignore: [429]).run
+  proofer = HTMLProofer.check_directory(
+    './_site/',
+    check_html: true,
+    hydra: { max_concurrency: 10 },
+    url_ignore: [%r{https://developer.github.com}, %r{https://docs.github.com}, %r{https://help.github.com}],
+    http_status_ignore: [429])
+  proofer.before_request do |request|
+    request.options[:headers]['Authorization'] = "Bearer #{GITHUB_TOKEN}" if request.base_url == "https://github.com"
```

Comment:
I'm bad at ruby, but does it need to be something like `ENV["GITHUB_TOKEN"]`

Reply:
Me too, just noticed due to failing CI :)